<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="radioButton_aircraft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Aircraft</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_helicopter">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Helicopter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rresult_text">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_aircraft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Aircraft</string>
    </property>
    <widget class="QComboBox" name="aircraft_airport_chouse_box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Sheremetevo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vnukovo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Domodedovo</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Create aircraft</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>altitude</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="regional_aircraft_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Regional aircraft</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="aircraft_name_lineEdit">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="aircaft_speed_spinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>350</number>
     </property>
     <property name="maximum">
      <number>1200</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="aircraft_max_altitude_spinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2000</number>
     </property>
     <property name="maximum">
      <number>15000</number>
     </property>
     <property name="singleStep">
      <number>500</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="aircraft_passanger_capacit_spinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Passenger</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>capacity</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="aircraft_range_spinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Range</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>211</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Note!</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_helicopter">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>131</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Helicopter</string>
    </property>
    <widget class="QLineEdit" name="helicopter_lineEdit_speed">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="helicopter_lineEdit_capacity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="helicopter_lineEdit_crew_szie">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="helicopter_lineEdit_max_altitude">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Airport_chouse_box_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Sheremetevo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vnukovo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Domodedovo</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="search_radioButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="search_groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>151</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Search</string>
    </property>
    <widget class="QSpinBox" name="search_range_spinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter range</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>capacity</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="search_capacity_spinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>544</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
